apache_exporter.go

```diff
@@ -200,13 +228,41 @@ func (e *Exporter) collect(ch chan<- prometheus.Metric) error {
 			}
 
 			ch <- prometheus.MustNewConstMetric(e.kBytesTotal, prometheus.CounterValue, val)
+		case key == "CPULoad":
+			val, err := strconv.ParseFloat(v, 64)
+			if err != nil {
+				return err
+			}
+
+			ch <- prometheus.MustNewConstMetric(e.cpuload, prometheus.CounterValue, val)
```

Comment:
MustNewConstMetric is for metric that can NOT change. Please change this to a NewGauge.